plugins/woocommerce-admin/client/marketplace/components/search-results/search-results.tsx

```diff
@@ -2,13 +2,21 @@
  * External dependencies
  */
 import { useQuery } from '@woocommerce/navigation';
+import { useContext } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import './search-results.scss';
 import { Product, ProductType, SearchResultType } from '../product-list/types';
 import Products from '../products/products';
+import NoResults from '../product-list-content/no-results';
+import { MarketplaceContext } from '../../contexts/marketplace-context';
+import {
+	MARKETPLACE_ITEMS_PER_PAGE,
+	MARKETPLACE_SEARCH_RESULTS_PER_PAGE,
+} from '~/marketplace/components/constants';
```

Comment:
Bero's VSCode was generating these dodgy paths, I think – maybe the same thing is happening with yours.

```suggestion
import {
	MARKETPLACE_ITEMS_PER_PAGE,
	MARKETPLACE_SEARCH_RESULTS_PER_PAGE,
} from '../../components/constants';
```

Reply:
Ahh, thanks! I won't accept it over GitHub but will do a commit with you as the co-author.